docs/scripting/trust scripts/sys.status.mdx

```diff
@@ -8,16 +8,34 @@ sys.status returns the status of the user that runs it.
 
 ## Syntax
 
+On the Command Line Interface:
+
+```
 sys.status
+```
+
+As a subscript call:
+
+```
+#fs.sys.status()
+```
 
 ### Parameters
 
-There are no known parameters
+There are no known parameters.
 
 ### Return
 
-It returns status, i need to get an actual output
+In the main terminal window, sys.status returns one of the following:
```

Comment:
```suggestion
sys.status returns a string in the CLI or an object in a script.

#### CLI

`-system status nominal-`

#### Script

```
{
  hardline: 0,
  tutorial: "",
  breach: false
}```

```

Reply:
This may make things much worse but it's intended to bring it inline with the template